notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>General comment: Incorrect name for the SDK. It's "Explainable AI SDK" - please correct this throughout both files.</p><p> </p><p>Since you're making similar changes to both the image and the tabular notebook, I've just reviewed the image notebook. I added specific updates for this notebook, and also general comments. Please apply my general comments to the tabular notebook as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>